src/scss/cdb-components/buttons.scss

```diff
@@ -0,0 +1,164 @@
+// Buttons styles
+// ----------------------------------------------
+
+/* SG
+# Buttons/Primary
+
+```
+<button class="CDB-Button CDB-Button--primary is-medium">
+  <span class="CDB-Button-Text Text is-semibold Size-medium">SAVE</span>
+</button>
+
+<button class="CDB-Button CDB-Button--primary is-disabled">
+  <span class="CDB-Button-Text Text is-semibold Size-small">SAVE</span>
+</button>
+
+<button class="CDB-Button CDB-Button--primary is-mini">
+  <span class="CDB-Button-Text Text is-semibold Size-small">SAVE</span>
+</button>
+
+<button class="CDB-Button CDB-Button--primary is-mini is-disabled">
+  <span class="CDB-Button-Text Text is-semibold Size-small">SAVE</span>
+</button>
+
+<br/>
+<br/>
+<br/>
+
+<div style="background: #2E3C43; padding: 20px;">
+  <button class="CDB-Button CDB-Button--primary is-medium">
+    <span class="CDB-Button-Text Text is-semibold Size-medium">SAVE</span>
+  </button>
+
+  <button class="CDB-Button CDB-Button--primary is-disabled">
+    <span class="CDB-Button-Text Text is-semibold Size-small">SAVE</span>
+  </button>
+
+  <button class="CDB-Button CDB-Button--primary is-mini">
+    <span class="CDB-Button-Text Text is-semibold Size-small">SAVE</span>
+  </button>
+
+  <button class="CDB-Button CDB-Button--primary is-mini is-disabled">
+    <span class="CDB-Button-Text Text is-semibold Size-small">SAVE</span>
+  </button>
+</div>
+
+```
+*/
+
+@import '../cdb-utilities/mixins';
```

Comment:
If we are not using any mixin we shouldn't import it. If we need it in the future let's face it at that time.